develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/01-adapting-to-liferay-7s-api.markdown

```diff
@@ -2,8 +2,8 @@
 
 +$$$
 
-**Important:** Liferay @ide@'s Code Upgrade Tool is not officially released and
-is unsupported. It's available in @ide@ 3.1 milestone versions only.
+**Important:** Liferay IDE's Code Upgrade Tool is not officially released and
```

Comment:
@sez11a yeah, I did. I'll just re-add the tokens in for now. I've gotten mixed signals, but have not received confirmation (I've have had a hard time communicating due to the time difference). We can merge this PR (since it has other changes too), and once I get confirmation, I can resend removed tokens, if necessary.